src/core/a-animation.js

```diff
@@ -356,13 +357,14 @@ function getAnimationValues (el, attribute, dataFrom, dataTo, currentValue) {
   var from = {};
   var partialSetAttribute;
   var to = {};
-
-  if (attributeSplit.length === 2) {
+  if (attributeSplit.length === 2 && attributeSplit.indexOf('color') === -1) {
```

Comment:
I'd ignore everything about mappings. That's part of what is currently not core code (e.g., `<a-box>`). Though it should be able to interpolate attributes that it detects are colors (e.g., `color="#FFFFFF"`, but that doesn't require looking up schema.